libs/core/src/lib/calendar/calendar.component.spec.ts

```diff
@@ -1,21 +1,24 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { FdDatetimeModule, FdDate } from '../datetime';
 
 import { CalendarComponent } from './calendar.component';
 import { CalendarModule } from './calendar.module';
-import { FdDate } from './models/fd-date';
 
 describe('CalendarComponent', () => {
-    let component: CalendarComponent;
-    let fixture: ComponentFixture<CalendarComponent>;
+    let component: CalendarComponent<FdDate>;
+    let fixture: ComponentFixture<CalendarComponent<FdDate>>;
 
-    beforeEach(async(() => {
-        TestBed.configureTestingModule({
-            imports: [CalendarModule]
-        }).compileComponents();
-    }));
+    beforeEach(
+        waitForAsync(() => {
```

Comment:
Question out of scope from this PR - Is it worth to migrate `async` to `waitForAsync` for every unit test?

Reply:
Yes, it worths cause `async` is deprecated and renamed to `waitForAsync` ([async doc](https://angular.io/api/core/testing/async))